Makefile.psp.mk

```
@@ -39,7 +39,7 @@ $(BUILD_DIR)/tt_%.elf: $(BUILD_DIR)/tt_%.ld $$(call get_o_files,servant/tt_%) $(
 
 # Step 3/5 of build
 $(BUILD_DIR)/%.bin: $(BUILD_DIR)/$$(call add_ovl_prefix,%).elf
-	$(OBJCOPY) -O binary $< $@
+	$(if $(VERBOSE),,@echo "Copying objects from $*.elf to $(@:$(BUILD_DIR)/%=%)";) $(OBJCOPY) -O binary $< $@
```

```suggestion
	$(if $(VERBOSE),,@echo "Stripping $<";) $(OBJCOPY) -O binary $< $@
```
Let's keep it simple like the others

---

Thanks you!  You have no idea how nuts that was driving me because I couldn't find the right word for it.